pkg/common/schema/job.go

```diff
@@ -63,9 +63,11 @@ const (
 	EnvJobWorkerCommand   = "PF_JOB_WORKER_COMMAND"
 
 	// EnvJobModeCollective env
-	EnvJobModeCollective = "Collective"
-	EnvJobReplicas       = "PF_JOB_REPLICAS"
-	EnvJobFlavour        = "PF_JOB_FLAVOUR"
+	EnvJobModeCollective   = "Collective"
+	EnvJobReplicas         = "PF_JOB_REPLICAS"
+	EnvJobFlavour          = "PF_JOB_FLAVOUR"
+	EnvJobLimitFlavour     = "PF_JOB_LIMIT_FLAVOUR"
+	EnvJobLimitFlavourNone = "None"
```

Comment:
It might be better to make it case insensitive.

Reply:
done